src/templates/utils.js

```diff
@@ -0,0 +1,36 @@
+import tcomb from 'tcomb-validation';
+
+function toGetter(prop) {
+  const capsProp = prop.charAt(0).toUpperCase() + prop.slice(1);
```

Comment:
So annoying that Javascript doesn't have a titlecase method built-in... -_-

Reply:
agreed